### PR TITLE
updating about page

### PR DESCRIPTION
**Pull request by ramirezg**
Fixed some of the spacing and now updating some of the language to make to clearer.

### CODE REVIEW

File: foia_hub/templates/about.html [hunk 1]
```diff
@@ -20,6 +20,7 @@
             </p>
 
             <h2>Features</h2>
+            <p></p>
```
Comment (khandelwal) [inline foia_hub/templates/about.html]: Why do we have an empty paragraph here?

Comment (ramirezg) [inline foia_hub/templates/about.html]: It fixes the line spacing problem via https://github.com/18F/foia-hub/issues/436 I guess we could also use a line break.

Comment (khandelwal) [inline foia_hub/templates/about.html]: I don't know but I think that's not how that should be laid out. @vz3 can clarify.

**Comment by khandelwal** [comment]
I don't think using an empty paragraph to fix a spacing issue is the right thing. @vz3 will you take a look at this?

**Comment by vz3** [comment]
Layout changes here are addressed in #456. Closing.

**Comment by ramirezg** [comment]
:+1:

**Pull request closed without merging**